src/check/parse/tokenize.zig

```diff
@@ -5,12 +5,16 @@ const GenCatData = @import("GenCatData");
 
 pub const Token = struct {
     tag: Tag,
-    offset: u32,
-    length: u32,
+    offset: Index,
+    length: Index,
+
+    pub const Index = u32;
 
     pub const Tag = enum(u8) {
         EndOfFile,
+        // Whitespace
         Newline,
+        Comment,
```

Comment:
I don't _think_ this should be necessary? (maybe I missed your reasoning here...)

Reply:
Sorry, I meant to roll this back after our conversation.  I'll remove